### PR TITLE
[7437] Updating Readme files related to the last syscollector work

### LINKED ISSUE #7437: Wazuh in project documentation
## Description

This issue aims to move the readme from the RTR src / readme, also to update the readme from the RTR and move it to where is the build.py

Update with reliable information on the installation steps in INSTALL.MD in root project folder.

## DoD
- [x] Modify readme/install files.
- [x] PR

### PR DESCRIPTION
**Pull request by tinachoB**
|Related issue|
|---|
|Closes #7437 |

## **Description**

This issue aims to move the readme from the RTR src / readme, also to update the readme from the RTR and move it to where is the build.py

Update with reliable information on the installation steps in INSTALL.MD in root project folder.

## **Affected files**
- https://github.com/wazuh/wazuh/blob/7437-readmes-docs-update/INSTALL
- https://github.com/wazuh/wazuh/blob/7437-readmes-docs-update/src/Readme.md
- https://github.com/wazuh/wazuh/blob/7437-readmes-docs-update/src/data_provider/testtool/Readme.md
- https://github.com/wazuh/wazuh/blob/7437-readmes-docs-update/src/shared_modules/dbsync/testtool/Readme.md
- https://github.com/wazuh/wazuh/blob/7437-readmes-docs-update/src/wazuh_modules/syscollector/testtool/Readme.md

## **DoD**
- [X] Modify readme/install files.

**Before**
image

**After**
image

### CODE REVIEW

**Review by konangc87 — APPROVED** [review]

File: src/data_provider/testtool/Readme.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+2. [Compile Wazuh](#compile-wazuh)
+3. [How to use the tool](#how-to-use-the-tool)
+
+## Purpose
+The Data Provider Testing Tool was created to test and validate the data obtained by the module's execution. This tool works as a black box where an user will be able execute it and analyze the output data as desired.
+
+## Compile Wazuh
+In order compile the solution on a specific wazuh target, the project needs to be built either in release or debug/test mode.
+```
+make TARGET=server|agent <DEBUG=1 TEST=1>
```
Comment (Dwordcito) [inline src/data_provider/testtool/Readme.md:13]: Remove TEST option from this file, the test tool not are compiled in this mode.

File: src/wazuh_modules/syscollector/testtool/Readme.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+2. [Compile Wazuh](#compile-wazuh)
+3. [How to use the tool](#how-to-use-the-tool)
+
+## Purpose
+The Syscollector Testing Tool was created to test and validate the data obtained by the module's execution. This tool works as a black box where an user will be able execute it and analyze the output data as desired.
+
+## Compile Wazuh
+In order compile the solution on a specific wazuh target, the project needs to be built either in release or debug/test mode.
+```
+make TARGET=server|agent <DEBUG=1 TEST=1>
```
Comment (Dwordcito) [inline src/wazuh_modules/syscollector/testtool/Readme.md:13]: Remove TEST option from this file, the test tool not are compiled in this mode.

**Review by Dwordcito — APPROVED** [review]

File: src/wazuh_modules/syscollector/testtool/Readme.md (hunk 2 above)
Comment (tinachoB) [reply src/wazuh_modules/syscollector/testtool/Readme.md]: Done in last commit.

File: src/data_provider/testtool/Readme.md (hunk 1 above)
Comment (tinachoB) [reply src/data_provider/testtool/Readme.md]: Done in last commit.

**Pull request merged**